web/modules/custom/dpl_favorites_list/src/Controller/DplFavoritesListController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\dpl_favorites_list\Controller;
+
+use Drupal\Core\Block\BlockManagerInterface;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Render\RendererInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+/**
+ * Render favorites list react app.
+ */
+class DplFavoritesListController extends ControllerBase {
+
+  /**
+   * DplFavoritesListController constructor.
+   *
+   * @param \Drupal\Core\Block\BlockManagerInterface $blockManager
+   *   Drupal block manager.
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   Drupal renderer service.
+   */
+  public function __construct(
+    private BlockManagerInterface $blockManager,
+    private RendererInterface $renderer
+  ) {
+  }
```

Comment:
Is it on purpose that the constructor is empty? If yes please explain in a comment why.

Reply:
Constructor Property Promotion is a new syntax in PHP 8. So there is nothing to comment on here.

Reply:
Ah I se 👍